src/main/scala/com/gu/commercial/branding/Branding.scala

```diff
@@ -49,4 +51,6 @@ object Branding {
   * Signifies a paid container with multiple sponsors.
   * The container won't show a logo but will have the paid styling.
   */
-case object PaidMultiSponsorBranding extends ContainerBranding
+case object PaidMultiSponsorBranding extends HasBrandingType {
+  val brandingType = PaidContent
+}
```

Comment:
The way I'm returning this in MAPI is with a specific type for PaidMultiSponsorBranding so that the apps can tell the difference between a PaidMultiSponsorBranding and a regular PaidContent. 

If both can return the same type then I would have to return another field or do some some (potentially fragile) extra checks to determine if it is a simple branding or a multiSponsor one from the json representation.